### ISSUE TITLE
Desktop Wallet download not working

### ISSUE THREAD

**Issue by ThibaultJanBeyer** [amjuarez/bytecoin#164]
Hello, I am not able to download the desktop wallet for bytecoin...
Everytime I klick, I get redirected to https://bytecoin.org/downloads# without downloading anything.

Tried it on Firefox 57.0b9 (64-bit) on OSX
and on Chrome Version 61.0.3163.100 (Official Build) (64-bit) also on OSX

got the same result...
Please fix.

**Comment by spidertrack** [comment]
They are scam, I haven't been able to sync my wallet for months and after I complained with them my wallet is not loadng anymore and I have not saved the wallet because it was not synced this mean I have been conned and lost my coins? they are stealers.

**Comment by System96** [comment]
Spidertrack, quit spreading FUD. If you haven't been able to sync your wallet for months, you probably should have asked around for people to help you out. Anyway, here's a guide to syncing your wallet:
If you dont wait to manually wait, you can import the blockchain by downloading it and block indexes on bytecoin.org/downloads/, then import them by going into %appdata% and putting both the blockchain and block indexes into the folder. Then, it may take a couple hours to go through all of those blocks, but your wallet will be synced much faster. alternatively, you can just sync to a node by going to preferences then node, here is a list of nodes you can connect to:
67.205.156.172:8080
node.bytecoin.ninja:8080
51.15.76.100:8080
154.0.169.48:8080
13.58.214.223:8080
pool.go.ro:8080

Also, you can still backup your wallet while the wallet is syncing, and if you don't have access to that wallet anymore its your fault you didn't back it up. **You should never send coins to a wallet you don't have backed up, otherwise you are completely liable to losing your coins. Don't blame your ignorance on the dev team.**

**Comment by spidertrack** [comment]
Bytecoin is a scam.

**Comment by System96** [comment]
Great detective work there bud, first of all keep that off the Github page, this is for software errors not FUD.
And if your referring to the Cryptonote/Bytecoin scam post on BTC talk, then read the paper I wrote about that whole thing along with some other posts about relevant topics, which you can read here:
https://www.docdroid.net/rEKgQUR/bytecoin-rebuttle.pdf

TLDR: Nearly all of the information spread around Cryptonote related currency regarding them being scams are not backed by facts, and are just claims that are simply that. Claims.

And if your calling Bytecoin a scam because you lost your coins, then I'm afraid there's no hope for you. **Once again, don't blame the dev team for your incompetence and ignorance of how cryptocurrencys work. All of the losses you sustained are only your fault because you didn't do basic cryptocurrency rules and didnt back up your .wallet file.**

**Comment by ThibaultJanBeyer** [comment]
is this still actively developed?
It seems like a one-man show. Since 2014 only Antonio Juarez is committing stuff. The project has only 5 contributors and they do not work on it anymore. We have 8 open pull requests and 102 open issues.

**Comment by System96** [comment]
Bytecoin indeed is still being developed, something new software wise is coming on December 15th and the community is more active than ever, also about the one man show thing, We have been told that there are mutliple devs, but as for commits on only one account they could all be using a central account, judging by how they keep themselves very anonymous.
Also, some commits are pretty big for a one man show like this one: https://github.com/amjuarez/bytecoin/commit/4363a9f1001893c80ee2435399836cfe43b3014e
But who really knows 0_o

**Comment by spidertrack** [comment]
How much are you getting paid to back up bytecoin your dipshit? It is a
scam, no other currency steal your coins the way only bytecoin does I will
wait and it is but and I will sue the company. You are such an ass hole
buddy because everybody knows BYTECOIN is a SCAM.

**Comment by System96** [comment]
Spidertrack your comment has so many logical fallacies its killing my brain cells. First of all,I'm evidently not getting paid enough by the Bytecoin devs because I'm still on Medicaid :P 
Anyways your coins wont stolen, **you were just a complete idiot and sent coins to a wallet you didn't back up.** If there was a backend for the Bytecoin devs to steal peoples coins it would have been called out by now because **the bytecoin software is OPEN SOURCE.** Also, Bytecoin isn't a company and you can't sue them because you were ignorant on how you should store coins. You also resort to name calling again, which is also very telling.

Admit it Spidertrack, ** you were a total idiot and haphazardly sent coins to a wallet you didn't back up, and you can't swallow your pride and own up to the fact that you fucked up. **
Hell, you don't even know how to use Github, because you made Github repositorys ASKING HOW TO FIX THE ERROR YOUR HAVING. 
https://github.com/spidertrack/BYTECOIN-LOAD-ERROR-AND-NO-SAVED-WALLET-

**Dude,just admit that you fucked up and lost your coins, your making yourself look like a complete and utter idiot.**

**Comment by ThibaultJanBeyer** [comment]
You two? You sound like me when I was 13 years of age.
So I’ll do us a favour and close this thread.
Honestly, I don’t think that Bytecoin is still being developed. Otherwise, this would have been closed and fixed 23 days ago…
You can go on and continue bashing each other, I’m out of here, have a nice one and thanks for the funny „discussion“ 🍡

**Comment by spidertrack** [comment]
Come on dipshit don’t be angry because the con company you are working with
“Bytecoin” will have loads of lawsuits .Time will tell.

**Comment by spidertrack** [comment]
Yeah dipshit they stole my coin I haven’t lost it , bytecoin is a scam
everybody knows that .How much are you getting paid dipshit?

**Comment by System96** [comment]
I'm going to stop replying to your echo chamber Spidertrack unless you can prove that
A. I'm getting paid by Bytecoin
B. There's a Bytecoin "company"
C. That the devs have the ability to steal your coins

**Comment by aralsoft** [comment]
spidertrack, where is your evidence that Bytecoin is a scam ? "Everybody knows" argument does not count as evidence. I don't think it is a scam. Why would anyone steal Bytecoins ? it is still easy to mine. You sound like a lame crybaby. Anyway I am muting this thread.

**Comment by spidertrack** [comment]
Yeah dip shit do it stealer

**Comment by noglance** [comment]
spidertrack, how much You lost? What if I will send you this (same amount) funds back?

**Comment by spidertrack** [comment]
How will send it if my wallet doesn’t work on my computer anymore? Bytecoin
is a scam they totally took all and I can’t load anything .They stole
700,000

**Comment by System96** [comment]
Spidertrack, I'm breaking my promise of not responding until you proved my point, but I have to say this.

Man, **Someone LITERALLY OFFERED YOU TO GIVE YOU COMPENSATION FOR THE LOSSES YOU SUSTAINED AND YOU CONTINUE TO MAKE A FOOL OUT OF YOURSELF. ** It's called a web wallet: www.bytecoin.money

Spidertrack, either your a troll or your trying to put Bytecoin down because your salty because you spent nearly 1K USD **and didn't do proper research on how to keep your coins safe (if your spending 1K USD on someone DO YOUR FUCKING RESEARCH so you dont lose it!)**

Either way, all of your losses are your own fault. You were stupid and sent 1K of USD to a not backed up wallet and lost all of it.

**Comment by spidertrack** [comment]
Shut up dipshit you are still talking nonsense your bytecoin scam seller.

**Comment by spidertrack** [comment]
No dipshit only BYTECOIN is a scam , the other coins are not they did a
scam program exactly to scam people and this is why it doesn’t work anymore
and this is how they get to steal people’s money. They haven’t given my
coins back they stole it and it is a scam I am already writing a blog to
warn people I am hate scammers and bytecoin is a big scam just like you
dipshit.

**Comment by System96** [comment]
Have fun with your blog, also the BCN devs dont have your coins, no one has your coins because you **didn't back up your .wallet file.**

To all who are reading this, take a lesson from Spidertrack and make sure to backup your wallet before sending ANY coins to it! You don't want to end up a salty mess like Spidertrack!

**Comment by spidertrack** [comment]
Yeah dip shit keep conning people with your bytecoin fraud, this company is
a fraud. Be careful whoever is reading this.
